pkg/datarepair/checker/checker_test.go

```diff
@@ -132,6 +132,29 @@ func TestIdentifyIrreparableSegments(t *testing.T) {
 		// check if repair attempt count was incremented
 		require.Equal(t, 2, int(remoteSegmentInfo.RepairAttemptCount))
 		require.True(t, firstRepair < remoteSegmentInfo.LastRepairAttempt)
+
+		// make the  pointer repairable
+		pointer = &pb.Pointer{
+			Remote: &pb.RemoteSegment{
+				Redundancy: &pb.RedundancyScheme{
+					MinReq:          int32(3),
+					RepairThreshold: int32(8),
+				},
+				RootPieceId:  teststorj.PieceIDFromString("fake-piece-id"),
+				RemotePieces: pieces,
+			},
+		}
+		// put test pointer to db
+		metainfo = planet.Satellites[0].Metainfo.Service
+		err = metainfo.Put("fake-piece-id", pointer)
+		require.NoError(t, err)
+
+		// this deletes the previously entered irreparable pointer from irrdb
```

Comment:
When I first read this comment, I was like "well that certainly requires a comment!"  That's usually not a good sign.  Consider renaming IdentifyInjuredSegments.  Or maybe go for efficiency if you could call `checkSegmentStatus()` or `irrdb.Delete()` directly.

Reply:
I added the comment while I was doing development for debug thru the fmt.. It shouldnt be there ... removed the comment.